adapters/adorsys-integ-adapter/src/test/resources/mappings/v1_consents_lwigccdnqiju2wxmn2qjiiiwdpjrci55c92nupa5ixrng8jyetrvnjadwuefv7g4xcx1qlgcjausajiqoazagmz9mpajiqih3njx8ihgetw__psglvqpt9q_authorisations_19636dbe-99db-4d0f-90de-2573057bc7e4-76a06747-a112-4e07-b120-1cc2f61d5ee4.json

```diff
@@ -0,0 +1,46 @@
+{
+  "id" : "76a06747-a112-4e07-b120-1cc2f61d5ee4",
+  "name" : "v1_consents_lwigccdnqiju2wxmn2qjiiiwdpjrci55c92nupa5ixrng8jyetrvnjadwuefv7g4xcx1qlgcjausajiqoazagmz9mpajiqih3njx8ihgetw__psglvqpt9q_authorisations_19636dbe-99db-4d0f-90de-2573057bc7e4",
```

Comment:
it would be nice to have meaningful names of mapping files

Reply:
renamed